src/component/selection/getRangeClientRects.js

```diff
@@ -55,6 +56,7 @@ function getRangeClientRectsChrome(range: Range): Array<ClientRect> {
     'Found an unexpected detached subtree when getting range client rects.'
   );
 }
+/* eslint-enable consistent-return */
```

Comment:
Yup (see PR description 😛).

I just threw the `[]` in there & Flow complains that it's unreachable code because it does know about `invariant`.

I'll leave it as is.